lib/runtime.fast

```diff
@@ -4,6 +4,7 @@ import lang/wasm;
 import lang/c;
 import lang/flow;
 import lang/rust;
+import string_utils;
 
```

Comment:
We should limit what is in runtime.fast to reduce deps

Reply:
gensym is moved to string_utils